tests/unit_tests/data_steward/cdr_cleaner/cleaning_rules/null_person_birthdate_test.py

```diff
@@ -0,0 +1,55 @@
+# Python imports
+import unittest
+
+from cdr_cleaner.cleaning_rules.null_person_birthdate import (
+    NullPersonBirthdate, NULL_DATE_QUERY)
+# Project imports
+from constants.bq_utils import WRITE_TRUNCATE
+from constants.cdr_cleaner import clean_cdr as clean_consts
+from common import PERSON
+
+
+class NullPersonBirthdateTest(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        print('**************************************************************')
+        print(cls.__name__)
+        print('**************************************************************')
+
+    def setUp(self):
+        self.project_id = 'foo_project'
+        self.dataset_id = 'bar_dataset'
+        self.sandbox_id = 'baz_sandbox'
+
+        self.rule_instance = NullPersonBirthdate(self.project_id,
+                                                 self.dataset_id,
+                                                 self.sandbox_id)
+
+        self.assertEqual(self.rule_instance.project_id, self.project_id)
+        self.assertEqual(self.rule_instance.dataset_id, self.dataset_id)
+        self.assertEqual(self.rule_instance.sandbox_dataset_id, self.sandbox_id)
+
+    def test_get_query_specs(self):
+        # Pre conditions
+        self.assertEqual(self.rule_instance.affected_datasets,
+                         [clean_consts.CONTROLLED_TIER_DEID])
+
+        #Test
+        results_list = self.rule_instance.get_query_specs()
+
+        # Post conditions
+        expected_list = [{
+            clean_consts.QUERY:
+                NULL_DATE_QUERY.render(project_id=self.project_id,
+                                       dataset_id=self.dataset_id,
+                                       person_table=PERSON),
+            clean_consts.DESTINATION_TABLE:
+                PERSON,
+            clean_consts.DESTINATION_DATASET:
+                self.dataset_id,
+            clean_consts.DISPOSITION:
+                'WRITE_TRUNCATE'
```

Comment:
You can use the imported `WRITE_TRUNCATE` statement here instead of the string 👍🏾